tests/api/v1/collectors/heartbeat.js

```diff
@@ -52,7 +52,7 @@ describe('tests/api/v1/collectors/heartbeat.js >', () => {
     afterEach(u.forceDelete);
     after(tu.forceDeleteUser);
 
-    it('config ok', (done) => {
+    it.skip('config ok', (done) => {
       api.post(`/v1/collectors/${u.toCreate.name}/heartbeat`)
```

Comment:
is skipping this test intentional?

Reply:
Yes @annyhe pls. see my comment on this PR from 21hrs ago.